classes/mender-install.bbclass

```diff
@@ -1,6 +1,10 @@
 PREFERRED_VERSION_go_cross = "1.6%"
 
-#Add support for systemd
+# Add support for systemd
+#
+# Note that a very common error with systemd is that the system hangs during the
+# boot process while looking for devices. This is almost always because the
+# kernel feature CONFIG_FHANDLE is not enabled.
 DISTRO_FEATURES_append = " systemd"
```

Comment:
Well, it's not like we depend on a whole distro, we just depend on certain features of the existing distros. What would be the appropriate way, produce error if it's not included?